common/script/compiler.go

```diff
@@ -17,7 +17,7 @@ func CompileTypeScript(entrypoint string) (string, error) {
 		Platform:    api.PlatformNode,
 		GlobalName:  "exports",
 		EntryPoints: []string{entrypoint},
-		Sourcemap:   api.SourceMapInline,
+		Sourcemap:   api.SourceMapNone,
```

Comment:
Would we have a better error messages with inline source maps?

Reply:
Yep definitly, rolled back this change on the latest commit